ovn-tester/ovn_tester.py

```diff
@@ -36,7 +37,9 @@ def usage(name):
     )
 
 
-def read_physical_deployment(deployment, global_cfg):
+def read_physical_deployment(
+    deployment: str, global_cfg: GlobalCfg
+) -> Tuple[PhysicalNode, List[PhysicalNode]]:
```

Comment:
This actually returns `Tuple[List[PhysicalNode], List[PhysicalNode]]`.

Reply:
done